### PR TITLE
Task6

### PR DESCRIPTION
**Pull request by sonic-interlock**
1 - File serverless.yml contains configuration for catalogBatchProcess function
2 - File serverless.yml contains policies to allow lambda catalogBatchProcess function to interact with SNS and SQS
3 - File serverless.yml contains configuration for SQS catalogItemsQueue
4 - File serverless.yml contains configuration for SNS Topic createProductTopic and email subscription

No optional tasks

FE app url https://d1xz8rgl6jxoup.cloudfront.net/

Example of a csv file
```
title,description,price
"Lion","Sand cat",10
"Tiger","Tabby cat",20
"Panther","Black cat",30
```

### CODE REVIEW

File: import-service/handler.js [hunk 1]
```diff
@@ -33,19 +33,38 @@ module.exports.importProductsFile = async event => {
...
+          )
+          allData.push(parsed);
+        });
         stream.on('error', err => rej(err));
         stream.on('end', () => res())
-    });
+    })
+
+    await dataParsed;
+    await sns.publish({
```
Comment (ahordzin) [inline import-service/handler.js:60]: You need to move at least sns service (it better to move sns and sqs services) to product-service and send emails only after products are created

**Comment by ahordzin** [comment]
For me API doesn't work (I've filled csv file with data you've provided) - I receive Forbidden error in console. 
Try to use new S3({ region: "eu-west-1", **signatureVersion: "v4"** }) in importProducts handler. It should resolve the issue.

**Comment by sonic-interlock** [comment]
> For me API doesn't work (I've filled csv file with data you've provided) - I receive Forbidden error in console.
> Try to use new S3({ region: "eu-west-1", **signatureVersion: "v4"** }) in importProducts handler. It should resolve the issue.

Thank you!

File: import-service/handler.js (hunk 1 above)
Comment (sonic-interlock) [reply import-service/handler.js:60]: Thanks
Yeah, I understand why it is better to do in this way, but I'm not going to do it until Sunday

**Pull request merged**